modules/getting-started-cli-granting-permissions.adoc

```diff
@@ -6,20 +6,21 @@
 [id="getting-started-cli-granting-permissions_{context}"]
 = Granting view permissions
 
-{product-title} automatically creates a few special service accounts in every project. The default service account takes responsibility for running the pods. {product-title} uses and injects this service account into every pod that launches.
+{product-title} automatically creates several service accounts in every project. The `default` service account takes responsibility for running the pods. {product-title} uses and injects this service account into every pod that launches.
```

Comment:
```suggestion
{product-title} automatically creates several service accounts in every project. The `default` service account runs the pods. {product-title} uses and injects this service account into every pod that launches.
```

Reply:
I'm not positive if this changes the meaning, so I think I'll leave it as is.